### PR TITLE
HL-312 | Fix skip-to-content texts

### PR DESCRIPTION
**Pull request by sirtawast**
## Description 

Fix skip-to-content links for handler and applicant.

image

### CODE REVIEW

File: frontend/benefit/handler/public/locales/sv/common.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
     },
     "languageMenuButtonAriaLabel": "Ändra språk",
     "logoutLabel": "Logga ut",
-    "loginLabel": "Logga in"
+    "loginLabel": "Logga in",
+    "userAriaLabelPrefix": "Käyttäjä:",
+    "linkSkipToContent": "Hyppää pääsisältöön",
+    "menuToggleAriaLabel": "Valikko"
```
Comment (mjturt) [inline frontend/benefit/handler/public/locales/sv/common.json:20]: Code style thing again :D If I correctly interpret our `.prettierrc`, we should always have trailing commas?

**Review by mjturt — CHANGES_REQUESTED** [review]
Code looks good, can't say anything about the Swedish stuff. One code style thing also.

File: frontend/benefit/applicant/public/locales/fi/common.json [hunk 2]
```diff
@@ -369,6 +369,7 @@
     "loginLabel": "Kirjaudu palveluun",
     "logoutLabel": "Kirjaudu ulos",
     "userAriaLabelPrefix": "Käyttäjä:",
+    "linkSkipToContent": "Hyppää pääsisältöön",
```
Comment (mjturt) [inline frontend/benefit/applicant/public/locales/fi/common.json:372]: Also, some other Helsinki projects seem to be using the phrasing "Siirry sivun pääsisältöön" and "Gå till huvudinnehåll".

File: frontend/benefit/handler/public/locales/sv/common.json (hunk 1 above)
Comment (sirtawast) [reply frontend/benefit/handler/public/locales/sv/common.json:20]: This is JSON, not JS. So no, it must be valid JSON.

File: frontend/benefit/applicant/public/locales/fi/common.json (hunk 2 above)
Comment (sirtawast) [reply frontend/benefit/applicant/public/locales/fi/common.json:372]: Just copied the texts from www.hel.fi 🤷  and didn't want to translate en/sv again...

File: frontend/benefit/handler/public/locales/sv/common.json (hunk 1 above)
Comment (sirtawast) [reply frontend/benefit/handler/public/locales/sv/common.json:20]: Locales other than fi is not used in handler at all. The test scaffolding however needs the locales to be 1:1 with the keys. So I've just filled in with placeholders wherever necessary.

Comment (mjturt) [reply frontend/benefit/handler/public/locales/sv/common.json:20]: Ah, sorry, my bad.

File: frontend/benefit/applicant/public/locales/fi/common.json (hunk 2 above)
Comment (mjturt) [reply frontend/benefit/applicant/public/locales/fi/common.json:372]: Ok, if it's there, then it's probably fine.

**Pull request merged**